src/platform-implementation-js/platform-implementation.ts

```diff
@@ -52,7 +52,13 @@ export type PiOpts = {
 };
 export class PlatformImplementation extends SafeEventEmitter {
   destroyed: boolean;
+  /**
+   * @internal
+   */
   LOADER_VERSION: string;
+  /**
+   * @internal
+   */
   IMPL_VERSION: string;
```

Comment:
These properties aren't meant to be private.

```suggestion
  LOADER_VERSION: string;
  IMPL_VERSION: string;
```